web/client/codechecker_client/credential_manager.py

```diff
@@ -64,33 +64,45 @@ def __init__(self):
         user_home = os.path.expanduser("~")
         session_cfg_file = os.path.join(user_home,
                                         ".codechecker.passwords.json")
-        LOG.debug(session_cfg_file)
+        LOG.info("Checking local passwords or tokens in %s", session_cfg_file)
 
         scfg_dict = {}
         if os.path.exists(session_cfg_file):
+            check_file_owner_rw(session_cfg_file)
             scfg_dict = load_json_or_empty(session_cfg_file, {},
                                            "user authentication")
             scfg_dict['credentials'] = \
                 simplify_credentials(scfg_dict['credentials'])
-        if os.path.exists(session_cfg_file):
-            check_file_owner_rw(session_cfg_file)
+
+            if not scfg_dict['credentials']:
+                LOG.info("No saved tokens.")
+            else:
+                LOG.info("Tokens or passwords were found for these hosts:")
+                for k, v in scfg_dict['credentials'].items():
+                    user, _ = v.split(":")
+                    LOG.info("  user '%s' host '%s'", user, k)
```

Comment:
Are you sure that we would like to print every credentials which is found in this file in INFO log level?
We can print entries what are related to the specified url parameter in INFO log level, other entries should be printed in the DEBUG log level.